tests/test_abstract_inst/test_optical_spectrum_analyzer.py

```diff
@@ -37,8 +31,8 @@ def osc(monkeypatch):
 def test_osa_channel(osa):
     """Get channel: ensure existence."""
     with expected_protocol(osa, [], []) as inst:
-        with pytest.raises(NotImplementedError):
-            _ = inst.channel
+        ch = inst.channel[0]
+        assert isinstance(ch, ik.abstract_instruments.OpticalSpectrumAnalyzer.Channel)
```

Comment:
the channel itself is automatically implemented now and - if a single channel instrument - will forward to the parent's channel. so this needed to be changed